pkg/metrics/server.go

```diff
@@ -11,6 +11,11 @@ import (
 	"k8s.io/klog/v2"
 )
 
+var (
+	tlsCRT = "/etc/secrets/tls.crt"
```

Comment:
how did you derive these locations ... a comment here explaining that would be good

Reply:
this is the location to which I mounted the secret volume in the https://github.com/openshift/jenkins-sync-plugin/pull/408 PR

Reply:
huh .... was does a jenkins sync plugin change have to do with shared resource csi driver?

Reply:
@gabemontero of course i meant my CSO PR: https://github.com/openshift/cluster-storage-operator/pull/243

Reply:
ok that make sense :-)

now add a comment in the source here referring to that mount being done in the driver's deployment (which is managed by the CSO)